badger/cmd/flatten.go

```diff
@@ -33,36 +35,48 @@ This command would compact all the LSM tables into one level.
 	RunE: flatten,
 }
 
-var keyPath string
-var numWorkers int
+var fo = struct {
+	keyPath         string
+	numWorkers      int
+	numVersions     int
+	compressionType uint32
+}{}
 
 func init() {
 	RootCmd.AddCommand(flattenCmd)
-	flattenCmd.Flags().IntVarP(&numWorkers, "num-workers", "w", 1,
+	flattenCmd.Flags().IntVarP(&fo.numWorkers, "num-workers", "w", 1,
 		"Number of concurrent compactors to run. More compactors would use more"+
 			" server resources to potentially achieve faster compactions.")
-	flattenCmd.Flags().IntVarP(&numVersions, "num_versions", "", 1,
+	flattenCmd.Flags().IntVarP(&fo.numVersions, "num_versions", "", 0,
```

Comment:
For some options we are using snake case and for some we are using kebab case. Can we make it consistent?

Reply:
We cannot change them now. That will be a breaking change.